src/main/java/ch/njol/skript/expressions/ExprInventorySlot.java

```diff
@@ -41,78 +32,87 @@
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.Kleenean;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.PlayerInventory;
+import org.jetbrains.annotations.Nullable;
+
 @Name("Inventory Slot")
-@Description({"Represents a slot in an inventory. It can be used to change the item in an inventory too."})
-@Examples({"if slot 0 of player is air:",
+@Description("Represents a slot in an inventory. It can be used to change the item in an inventory too.")
+@Examples({
+	"if slot 0 of player is air:",
 	"\tset slot 0 of player to 2 stones",
 	"\tremove 1 stone from slot 0 of player",
-	"\tadd 2 stones to slot 0 of player",
-	"\tclear slot 1 of player"})
-@Since("2.2-dev24")
+	"\tadd 2 stones to slot 0 of inventory of all players",
```

Comment:
```suggestion
	"\tset slot 0 of all players to stone",
```
you cannot add to a single slot

Reply:
You can
![image](https://github.com/SkriptLang/Skript/assets/10354869/0fda455d-1552-491b-8acc-ffb09b520c62)